velox/exec/HashBuild.cpp

```diff
@@ -548,17 +514,15 @@ void HashBuild::spillInput(const RowVectorPtr& input) {
   computeSpillPartitions(input);
 
   vector_size_t numSpillInputs = 0;
-  for (auto row = 0; row < numInput; ++row) {
-    const auto partition = spillPartitions_[row];
-    if (FOLLY_UNLIKELY(!activeRows_.isValid(row))) {
+  for (auto rowIdx = 0; rowIdx < numInput; ++rowIdx) {
```

Comment:
The other place also use row for this. Let's keep the original variable naming? Thanks!